src/txn/txn_rollback_to_stable.c

```diff
@@ -577,52 +573,72 @@ __rollback_abort_col_ondisk_kv(WT_SESSION_IMPL *session, WT_REF *ref, WT_COL *ci
     __wt_cell_unpack_kv(session, page->dsk, kcell, vpack);
     prepared = vpack->tw.prepare;
 
-    if (vpack->tw.durable_start_ts > rollback_timestamp ||
```

Comment:
This function is also the same as the row store function, Is it possible to combine it row store?

Reply:
Yeah good idea, I've combined them into just __rollback_abort_ondisk_kv